app/modules/vm/index.coffee

```diff
@@ -162,6 +163,8 @@ module.exports = angular.module 'xoWebApp.vm', [
         oVdi = get oVbd.VDI
         continue unless oVdi
         VDIs.push oVdi if oVdi and not oVbd.is_cd_drive
+        if (isNumber(oVdi.size))
+          oVdi.size = bytesToSizeFilter(oVdi.size)
```

Comment:
I agree, I can do it another way but I'll need more time. I don't think I can easily handle a format difference between internal model and display/typing, using xEditable as it is used now (in fact I hate this ready made edit form component).